src/components/ProjectCard.tsx

```
@@ -163,88 +170,156 @@ const ProjectCard: Component<ProjectCardProps> = (props) => {
                 >
-                    <div class={`absolute inset-0 ${CARD_BG_CLASS}`}></div>
-                    <div class="absolute inset-0 bg-[radial-gradient(ellipse_at_top_right,rgba(255,255,255,0.10),transparent_70%)] mix-blend-overlay"></div>
-
-                    <div class="absolute inset-0 z-0 bg-linear-to-b from-black/10 via-transparent to-black/60 transition-opacity duration-200 group-hover:opacity-90"></div>
+                    {/* ASCII pattern decoration */}
+                    <div
+                        class="pointer-events-none absolute top-3 right-4 font-mono text-xs tracking-tight select-none"
+                        style={{ color: 'rgba(255, 255, 255, 0.15)', 'letter-spacing': '0.05em' }}
+                        aria-hidden="true"
+                    >
+                        /////
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)